package.json

```diff
@@ -27,20 +27,22 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome": ">=1.2.0-5",
+    "@fortawesome/fontawesome-free-brands": "^5.1.0-3",
```

Comment:
By adding this as a dependency will this bloat the `vendors.js`?

Reply:
I think it would only bloat `vendors.js` if the `index.js` imports it into the `vendorTree`, which it doesn't appear to be doing currently. However, I suspect that the only reason for this dependency is for the dummy app to be able to show icons, in which case, this would be better in `devDependencies`, right? In fact, that's where it was in the previous revision.

Reply:
My thought here is that if you're not using pro pack and you are only using static icons you shouldn't need to add any additional packages to your project. Since this work is done at build time and never put into the `vendorTree` it wont' bloat the app, but it will provide the simplest possible experience for most users. 
Just do `ember install ember-fontawesome` and then go ahead and use 
`{{fa-icon 'coffee'}}` in your app.

My feeling is that once this is merged and working we could flip the default rollup behavior to exclude any icons not explicitly requested and needed for dynamic usage or transforms. So nothing will get added to the app unless it is needed, but everything other than pro can be available easily to use.